packages/tokens/src/index.ts

```diff
@@ -1,5 +1 @@
-export * from './colors';
-export * from './motion';
-export * from './shadows';
 export * from './spacing';
```

Comment:
```suggestion
export { spacing } from './spacing';
```
We discussed this a little on the call, but it would be nice to explicitly define the named exports at the root `index.ts` of each package.

There are two things I like about this approach:
- Easily catch naming conflicts (Although, you pointed out that TS flags these issues 👍 )
- Easily see everything that's exposed from the package

Thoughts?

Reply:
+1 this! 👍